api/src/pcapi/scripts/venue/main.py

```diff
@@ -0,0 +1,50 @@
+"""
+Job console documentation here: https://www.notion.so/passcultureapp/Documentation-Job-Console-769beeacd5a146de9c97b6f8ee544276
+Assumed path to the script (copy-paste in github actions):
+
+https://github.com/pass-culture/pass-culture-main/blob/bdalbianco/PC-37584_update_empty_venue_publicName/api/src/pcapi/scripts/venue/main.py
+
+"""
+
+import argparse
+import logging
+
+import sqlalchemy as sa
+
+from pcapi.app import app
+from pcapi.core.offerers import models as offerer_models
+from pcapi.models import db
+
+
+logger = logging.getLogger(__name__)
+
+
+def main(not_dry: bool) -> None:
+    venues = (
+        db.session.query(offerer_models.Venue)
+        .filter(sa.or_(offerer_models.Venue.publicName == '""', offerer_models.Venue.publicName == ""))
```

Comment:
Je lis dans le ticket « Lorsqu’on le nom public d’une Venue est null ou "" », ce qui selon ma compréhension correspondrait plutôt à : 
```suggestion
        .filter(sa.or_(offerer_models.Venue.publicName == "", offerer_models.Venue.publicName.is_(None)))
```
As-tu vraiment des guillemets dans les `Venue.publicName` en base de données ?